native/src/modules/feedback/FeedbackContainer.js

```diff
@@ -0,0 +1,167 @@
+// @flow
```

Comment:
What changes have been made here? Sadly there is no nice git diff :/

Reply:
You just moved stuff from `FeedbackModalContainer` here, right?

Reply:
Yeah, would be nice to have the diff in git... :(

Before my changes:
- FeedbackModalContainer had functions for sending feedback of positive/negative feedback
- SearchModalContainer had the functions for sending feedback of search feedback
- FeedbackModal had UI for positive/negative feedback
- NothingFoundFeedbackBox had the UI for search feedback

Now:
- FeedbackContainer has functions for every feedback (positve/negative and search)
- Feedback has UI for every feedback
- FeedbackModalContainer is the page with FeedbackContainer as a child
- SearchModal is the page for the Search including FeedbackContainer

FeedbackContainer and Feedback are moved to modules-folder because they are shared now.
Logic and UI only exists once for both. 

FeedbackContainer has the functions from FeedbackModal Container and
- the FeedbackOriginType is added, so FeedbackContainer can also be used for Search-Feedback
- Search_Route-case is added to getFeedbackType-function
- getFeedbackData() is adjusted to work with FeedbackOriginType

Logic about the "page-stuff", like connect, refresh, mapStateToProps, withPayloadProvider... still are in FeedbackModalContainer.

Reply:
Thanks for summarizing!